simple_api/object/validators.py

```diff
@@ -0,0 +1,73 @@
+from inspect import isclass
+
+
+def build_validation_fn(action_validators, field_validators):
+    """
+    Prepares function for validation of input values and the action overall, instantiates validators which are not
+    already instantiated. Action_validators can be a sigle validator, Field_validators a List of tuples
+    (string, (validators...)) as prepared in object/actions.py
+    """
+    if action_validators and field_validators is None:
+        return None
+
+    if not isinstance(action_validators, (list, tuple)):
+        action_validators = action_validators,
+
+    # Validators instantiation if they are not already
+    instantiated_validators = []
+    for cls_or_inst in action_validators:
+        if isclass(cls_or_inst):
+            instantiated_validators.append(cls_or_inst())
+        else:
+            instantiated_validators.append(cls_or_inst)
+    for tup_of_validators in field_validators:
+        for validator in tup_of_validators[1]:
+            if isclass(validator):
+                instantiated_validators.append((tup_of_validators[0], validator()))
+            else:
+                instantiated_validators.append((tup_of_validators[0], validator))
```

Comment:
toto se zmeni, az zmenis zpusob ulozeni validatoru do slovniku, ted v tom uprimne ani nevidim, co se to snazi delat